salt/metalk8s/kubernetes/etcd/init.sls

```diff
@@ -5,6 +5,7 @@
 # ================
 #
 # * installed     -> deploy etcd manifest
+# * health        -> check health of etcd node
```

Comment:
Not part of `installed` or `init` because for expansions cluster is NOT healthly just after the manifest creation (as we need to execute the runner to add an etcd node first)